src/components/FormValidator.js

```diff
@@ -54,10 +58,10 @@ export default class FormValidator {
   }
 
   _setEventListeners() {
-    this._inputEls = Array.from(
-      this._form.querySelectorAll(this._inputSelector)
-    );
-    this._submitButton = this._form.querySelector(this._submitButtonSelector);
+    //this._inputEls = Array.from(
+    //this._form.querySelectorAll(this._inputSelector)
+    //);
```

Comment:
Please, delete `this.disableButton();` from line 76 in `FormValidator`

You need to disable the button only in block `then` to be able to click it in case there is a server error